mmcls/structures/multi_task_data_sample.py

```diff
@@ -39,7 +39,14 @@ def set_gt_task(self, value: Dict) -> 'MultiTaskDataSample':
 
     def set_pred_task(self, value: Dict) -> 'MultiTaskDataSample':
         """Set score of ``pred_task``."""
-        self.pred_task = LabelData(**value)
+        if list(value.values())[0] == torch.tensor:
```

Comment:
maybe you should check the type, if not seen it means that it is not covered by any unit test, please cover this and fix this

Reply:
What happens in value is an empty dict ? please add a unit test, it should not fail, set self.pred_task to `LabelData`